docs/v1/P4Runtime-Spec.mdk

```diff
@@ -2574,24 +2575,27 @@ An `ActionProfileMember` entity update message has the following fields:
 * `action` is the specification of the P4 action instance bound to the action
   profile member entry.
 
-An action profle member may be inserted, modified or deleted as per the
+An action profile member may be inserted, modified or deleted as per the
 following semantics.
 
 * `INSERT`: Add a new member entry bound to an eligible P4 action
   specification. The member id must be different from ids of already programmed
-  entries for that extern. The action specification must be provided. The total
-  number of members should not exceed the maximum specified in the P4 extern
-  specification, as a result of this insertion.
+  entries for that extern, or the server must return an `ALREADY_EXISTS` error
+  code. The action specification must be provided or the server must return
+  `INVALID_ARGUMENT`. The total number of members should not exceed the maximum
+  specified in the P4 extern specification as a result of this insertion or the
+  server should return `RESOURCE_EXHAUSTED`.
```

Comment:
Style nit - there's a comma before "or the server must return" in one of the 3 INSERT error cases.   Do you want to make them consistent?

Reply:
Will fix (probably add a comma everywhere)